wsi_service_base_plugins/tifffile/wsi_service_plugin_tifffile/slide.py

```
@@ -9,34 +9,47 @@
-        # read pixel sizes from xml image description
-        try:
-            self.ome_metadata = self.tif_slide.ome_metadata
-            self.parsed_metadata = xml.fromstring(self.ome_metadata)
-        except Exception as ex:
-            raise HTTPException(status_code=400, detail=f"Could not obtain ome metadata ({ex})")
-        self.slide_info = self.__get_slide_info_ome_tif()
+
+        if self.tif_slide.is_ome:
+            try:
+                self.ome_metadata = self.tif_slide.ome_metadata
+                self.parsed_metadata = xml.fromstring(self.ome_metadata)
```

Ruff (0.15.15)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)